media/static-bundles.json

```diff
@@ -1065,6 +1065,17 @@
       ],
       "name": "experiment-variation-whatsnew-68"
     },
+    {
+      "files": [
+        "js/base/uitour-lib.js",
```

Comment:
No need to reinclude `uitour-libs.js` or `up-to-date.js` since they are already included in the `firefox_whatsnew_2019` bundle.